lib/appsignal/config.ex

```diff
@@ -390,9 +385,43 @@ defmodule Appsignal.Config do
 
   @log_filename "appsignal.log"
 
-  def log_file_path do
+  def log_level do
     config = Application.fetch_env!(:appsignal, :config)
-    do_log_file_path(config[:log_path])
+
+    log_level(config)
+  end
+
+  defp log_level(config) do
+    case config[:log_level] do
+      "trace" -> :trace
+      "debug" -> :debug
+      "info" -> :info
+      "warn" -> :warn
+      "error" -> :error
+      _ -> deprecated_log_level(config)
+    end
+  end
+
+  defp deprecated_log_level(config) do
+    cond do
+      config[:transaction_debug_mode] -> :trace
+      config[:debug] -> :debug
+      true -> :info
+    end
+  end
+
+  def log_file_path do
+    case Application.fetch_env(:appsignal, :"$log_file_path") do
```

Comment:
I'm not familiar with this naming. Is the `$` a way to indicate private keys in the application env? Similar to using an unscore `_`?

Reply:
That was the intent, to indicate it's private, as well as to avoid a potential collision with someone specifying `log_file_path` in their `config.exs`. I think I got the idea of using `$` from [the tracer, which names its ETS table `$appsignal_registry`](https://github.com/appsignal/appsignal-elixir/blob/709224ad62326f7ede0b86497fe9df87c3713a6d/lib/appsignal/tracer.ex#L10).

Reply:
Aha! That makes sense.